Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -138,59 +141,36 @@ - (BOOL)togglePause:(BOOL)pause forCall:(OCTCall *)call error:(NSError **)error
             return NO;
         }
 
-        [self.calls updateCall:call updateBlock:^(OCTCall *callToUpdate) {
-            callToUpdate.status = pause ? OCTCallStatusPaused : OCTCallStatusActive;
-        }];
-
+        switch (control) {
+            case OCTToxAVCallControlResume:
+                [self.calls updateCall:call updateBlock:^(OCTCall *callToUpdate) {
+                [callToUpdate startTimer];
+                callToUpdate.status = OCTCallStatusActive;
+            }];
```

Comment:
So go ahead and move that back to one line? I had that originally and uncrustify moved it back down in a seperate line after I ran 'apply'